### PR TITLE
Temp solution done.

### PR DESCRIPTION
**Pull request by aliaksandr-galgouski**

### CODE REVIEW

File: config/webpack.prod.config.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+var merge = require('webpack-merge');
```
Comment (AlexanderGolub) [inline config/webpack.prod.config.js]: `const` can be used everywhere

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+export default class App {
+  constructor() {
+    this.service = new NewsApiService(API_KEY);
+  }
+
+  async init() {
+    this.sourceId = this.getQueryStringValue("sourceId");
+
+    let sources = await this.service.getSources();
```
Comment (AlexanderGolub) [inline src/app.js]: should be `const` as there is no other assignment

File: src/app.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+  async init() {
+    this.sourceId = this.getQueryStringValue("sourceId");
+
+    let sources = await this.service.getSources();    
+    if (!this.sourceId.length && sources.length) {
+      this.sourceId = sources[0].id;
+    }
+
+    let articles = await this.service.getArticles(this.sourceId);
```
Comment (AlexanderGolub) [inline src/app.js]: should be `const`

File: src/app.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    this.populateElement("articlesList", articles, this.createArticleHtml);    
+  }
+
+  getQueryStringValue(parameter) {
+    let params = new URL(document.location).searchParams;
+    return params.get(parameter) || "";
+  }
+
+  populateElement(elementId, models, contentFactory) {
+    var element = document.getElementById(elementId);
```
Comment (AlexanderGolub) [inline src/app.js]: should be `const` or at least `let`

File: src/app.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    element.innerHTML = "";
+
+    if (Array.isArray(models) && contentFactory) {
+      models.forEach(model => element.innerHTML += contentFactory.call(this, model));
```
Comment (AlexanderGolub) [inline src/app.js]: you don't need to use `call` and bind `this` as you don't use `this` in `createSourceHtml ` or `createArticleHtml `

Comment (aliaksandr-galgouski) [reply src/app.js]: This is need to retrieve `this.sourceId` from current App obj.

File: src/app.js (hunk 4 above)
Comment (aliaksandr-galgouski) [reply src/app.js]: fixed

File: src/app.js (hunk 3 above)
Comment (aliaksandr-galgouski) [reply src/app.js]: fixed

File: src/app.js (hunk 2 above)
Comment (aliaksandr-galgouski) [reply src/app.js]: fixed

File: config/webpack.prod.config.js (hunk 1 above)
Comment (aliaksandr-galgouski) [reply config/webpack.prod.config.js]: fixed

**Pull request merged**